UI/PongWars/src/global.json

```diff
@@ -1,6 +1,6 @@
 {
   // To update the version of Uno please update the version of the Uno.Sdk here. See https://aka.platform.uno/upgrade-uno-packages for more information.
   "msbuild-sdks": {
-    "Uno.Sdk": "5.2.112"
+    "Uno.Sdk": "5.3.96"
```

Comment:
This PR will need to be rebased with latest and updated to latest 5.4 stable version
Related new issue: https://github.com/unoplatform/Uno.Samples/issues/828

Reply:
```suggestion
    "Uno.Sdk": "5.4.5"
```